5.6/apache/Dockerfile

```diff
@@ -50,5 +50,5 @@ RUN { \
     && a2enmod rewrite
 
 # Install composer
-COPY install-composer /install-composer
+COPY image-files /
```

Comment:
Hmm, not sure. Less files -> less layers. The commands are really simple. We can really condense these into 2 lines (much like it was with earlier install instructions). We hardly ever have to change it anymore.